lectures/lecture09/redirection/README.md

```diff
@@ -0,0 +1,88 @@
+# Redirection
+
+Here we'll try out the fantastically powerful technique of redirection.
+We can use this to chain commands together easily to make things run fast.
+Let's make sure we're in the right place-- `pwd` should end in `lecture09/redirection`.
+
+## Pipes
+
+Remember how we didn't want to cat the `sequence.gb` file because it was too big?
+Sometimes the output from a command is also very verbose and we'd like to be able to page through it.
+For example, `tree /` will make a tree of every single file on your file system, which may be in the millions.
+You don't want to wait for that to scroll past.
+
+So!
+Try this:
+
+    tree / | less
+
+Magic!
+We can now use our less keys to page through the output.
+
+What happened here is that we took the output flowing from `tree` and connected it using a "pipe" to `less`.
+The "pipe" operator is `|`.
+The general format is
+
+    command_producing_output | command_consuming_output
+
+Perhaps `command_consuming_output` also produces output, and we'd like to continue the chain.
+We can chain together an arbitrary number of commands like
+
+    command1 | command2 | command3 | command4
+
+etc.
+
+Now it's your turn.
+Use redirection to count the number of files in the `lecture09` directory.
+(Remember the various flags we used for `ls`, and the fact that you can give it an argument.)
```

Comment:
My gut would be to use something like `ls ../../lecture09/ | wc -l` but we haven't covered `wc` yet?

Reply:
`wc` was the "practicum" for the first README.